base/initdefs.jl

```diff
@@ -354,6 +354,7 @@ const atexit_hooks = Callable[
     () -> Filesystem.temp_cleanup_purge(force=true)
 ]
 const _atexit_hooks_lock = ReentrantLock()
+const _atexit_hooks_finished = Ref(false)
```

Comment:
```suggestion
const _atexit_hooks_finished::Bool = false
```

Reply:
Oh nice, i didn't know you could do that! Much nicer 👍